### PR TITLE
Fix Qt deprecation warnings

### LINKED ISSUE #4061: Deprecated code in CGAL/Qt
## Issue Details
In Qt 5.13, the function `matrix()` from the painter has been deprecated. The warning specifies that `transform()` should be use dinstead. Modifications should be made accordingly.

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):ArchLinux
* Compiler:CLang
* Release or debug mode: both
* CGAL version: 4.13
* Other libraries versions if used (Eigen, TBB, etc.): Qt 5.13

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
Replace the `matrix` calls with `worldTransform`
## Release Management

* Affected package(s):GraphicsView
* Issue(s) solved (if any): fix #4061

### CODE REVIEW

**Comment by lrineau** [comment]
https://travis-ci.org/CGAL/cgal/jobs/555748608#L5468
```
In file included from .../cgal/GraphicsView/demo/Spatial_searching_2/Spatial_searching_2.cpp:16:
.../cgal/GraphicsView/demo/Spatial_searching_2/../../include/CGAL/Qt/PointsInKdTreeGraphicsItem.h:165:16: error: 
      no matching constructor for initialization of 'Draw<Traits>'
  Draw<Traits> draw(painter, &matrix);
               ^    ~~~~~~~~~~~~~~~~
.../cgal/GraphicsView/demo/Spatial_searching_2/Spatial_searching_2.cpp:212:41: note: 
      in instantiation of member function
      'CGAL::Qt::PointsInKdTreeGraphicsItem<CGAL::Kd_tree<CGAL::Search_traits_2<CGAL::Epick>,
      CGAL::Sliding_midpoint<CGAL::Search_traits_2<CGAL::Epick>,
      CGAL::Plane_separator<double> >, CGAL::Boolean_tag<true> > >::paint'
      requested here
  this->graphicsView->setSceneRect(pgi->boundingRect());
                                        ^
.../cgal/GraphicsView/demo/Spatial_searching_2/../../include/CGAL/Qt/PointsInKdTreeGraphicsItem.h:63:5: note: 
      candidate constructor not viable: no known conversion from 'QTransform *'
      to 'QMatrix *' for 2nd argument
    Draw(QPainter* painter, QMatrix* matrix)
    ^
.../cgal/GraphicsView/demo/Spatial_searching_2/../../include/CGAL/Qt/PointsInKdTreeGraphicsItem.h:57:9: note: 
      candidate constructor (the implicit copy constructor) not viable: requires
      1 argument, but 2 were provided
  class Draw
        ^
1 error generated.
```

File: Polyhedron/demo/Polyhedron/MainWindow.cpp [hunk 1]
```diff
@@ -2171,10 +2171,17 @@ void MainWindow::resetHeader()
   sceneView->header()->setSectionResizeMode(Scene::RenderingModeColumn, QHeaderView::ResizeToContents);
   sceneView->header()->setSectionResizeMode(Scene::ABColumn, QHeaderView::Fixed);
   sceneView->header()->setSectionResizeMode(Scene::VisibleColumn, QHeaderView::Fixed);
+#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
+           sceneView->header()->resizeSection(Scene::ColorColumn, sceneView->header()->fontMetrics().horizontalAdvance("_#_"));
+           sceneView->resizeColumnToContents(Scene::RenderingModeColumn);
+           sceneView->header()->resizeSection(Scene::ABColumn, sceneView->header()->fontMetrics().horizontalAdvance(QString("_AB_")));
+           sceneView->header()->resizeSection(Scene::VisibleColumn, sceneView->header()->fontMetrics().horizontalAdvance(QString("_View_")));
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/MainWindow.cpp]: indentation

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-87.shtml through #7073

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-87.shtml through #4073

**Pull request merged**